examples/MutableMap/MutableMapExamples.dfy

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT
+*******************************************************************************/
+
+// RUN: %dafny /compile:0 "%s"
+
+include "../../src/MutableMap/MutableMap.dfy"
+
+module {:options "-functionSyntax:4"} MutableMapExamples {
+  import opened MutableMap
+
+  method Main() {
+    var m := new MutableMap<string,string>();
+    assert m.Keys() == {};
+    assert m.Values() == {};
+    assert m.Items() == {};
+    assert m.Size() == 0;
+    m.Put("testkey", "testvalue");
```

Comment:
readability nit: a newline before each (operation + asserts) block would be really nice. :)